configure.ac

```diff
@@ -978,7 +978,7 @@ AS_CASE([$host_os_gui], [cygwin* | mingw*], [
         THREADS="threads$EXEEXT"
     ])
 
-    AS_IF([test x$enable_wayland = xyes], [
+    AS_IF([test x$enable_wayland != xno], [
```

Comment:
Is this a null-change?
This seems to change "== yes"  into "!= no", which is the same thing?
Or am I missing the point here?

Reply:
This would be a null change if `enable_wayland` could have only 2 values (yes or no). But a 3rd situation is possible where `enable_wayland` is empty, arising when neither `--enable-wayland` nor `--disable-wayland` are present in the `configure` command line. The code in master treats this as  `--disable-wayland` whereas the PR has it treated as `--enable-wayland`.

Reply:
OK.
Might be useful to have a comment somewhere to remind folks this is not a binary option, but instead tertiary...
I will certainly forget!